admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -79,9 +79,7 @@
              title="{$smarty.capture.title|htmlspecialchars}"
              href="{$link->getAdminLink('AdminMaintenance')|escape:'html':'UTF-8'}"
           >
-            <i class="material-icons"
-              style="{if isset($maintenance_allow_admins) && $maintenance_allow_admins}color: var(--green);{/if}"
```

Comment:
@tblivet functional change

Reply:
Same, I'm not sure this is used anymore ith the Symfony layout
But we didn't clean the templates yet for safety, I wonder if we should clean them all before the final release 🤔 I guess it should be possible since they're not used anyway